WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -18,6 +18,7 @@ class ReaderTabViewController: UIViewController {
         self.makeReaderTabView = readerTabViewFactory
         super.init(nibName: nil, bundle: nil)
 
+        extendedLayoutIncludesOpaqueBars = true
```

Comment:
The reader scroll view seems to have incorrect insets. I'm leaving the comment in this file, but it's not necessarily because of the `extendedLayoutIncludesOpaqueBars`.

https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/1b0899f1-69f7-4e1b-983b-8f0407e10657


I also tested the behavior in `trunk` for comparison:

https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/8ff24908-5902-4c89-a1ed-8e8758685450